src/libpriv/rpmostree-core.c

```diff
@@ -878,6 +878,51 @@ checkout_pkg_metadata_by_nevra (RpmOstreeContext *self,
 }
 
 gboolean
+rpmostree_get_nevra_from_pkgcache (OstreeRepo  *repo,
```

Comment:
The caller of this will be checking for metadata v4 right?  Worth a comment if so.